{{cookiecutter.project_slug}}/README.md

```diff
@@ -27,9 +28,12 @@ cd build
 cmake -DCMAKE_BUILD_TYPE=Release ..
 cmake --build .
 ```
-{% if cookiecutter.doxygen == "Yes" %}
-# Documentation
 
+# Documentation
+{% if cookiecutter.readthedocs == "Yes" %}
+{{ cookiecutter.project_name }} provides a Sphinx-based documentation, that can
+be browsed [online at readthedocs.org](https://readthedocs.org).
```

Comment:
```suggestion
be browsed [online at readthedocs.org](https://{{ cookiecutter.project_slug }}.readthedocs.io).
```

Reply:
I had already realized that as well, so your patch does not apply anymore, but the content is on the branch.